pkg/commands/pull_request.go

```diff
@@ -131,9 +155,13 @@ func (pr *PullRequest) getPullRequestURL(branch *models.Branch) (string, error)
 	}
 
 	repoInfo := getRepoInfoFromURL(repoURL)
-	pullRequestURL := fmt.Sprintf(
-		gitService.PullRequestURL, repoInfo.Owner, repoInfo.Repository, branch.Name,
-	)
+	var toBranchName string
+	if to == nil {
+		toBranchName = ""
+	} else {
+		toBranchName = to.Name
+	}
```

Comment:
I think this is a bit cleaner
```suggestion
	toBranchName := ""
	if to != nil {
	        toBranchName = to.Name
	}
```

Reply:
I prefer the original form, but only marginally. I've used both forms throughout the code.